velocity/src/main/java/me/leoko/advancedban/velocity/VelocityMain.java

```diff
@@ -0,0 +1,62 @@
+package me.leoko.advancedban.velocity;
+
+import com.google.inject.Inject;
+import com.velocitypowered.api.event.proxy.ProxyInitializeEvent;
+import com.velocitypowered.api.event.Subscribe;
+import com.velocitypowered.api.event.proxy.ProxyShutdownEvent;
+import com.velocitypowered.api.plugin.Dependency;
+import com.velocitypowered.api.plugin.Plugin;
+import com.velocitypowered.api.plugin.annotation.DataDirectory;
+import com.velocitypowered.api.proxy.ProxyServer;
+import me.leoko.advancedban.Universal;
+import me.leoko.advancedban.velocity.listener.ChatListenerVelocity;
+import me.leoko.advancedban.velocity.listener.ConnectionListenerVelocity;
+import org.slf4j.Logger;
+
+import java.nio.file.Path;
+
+@Plugin(
+    id = "advancedban",
+    name = "AdvancedBan",
+    version = "2.3.0",
```

Comment:
@ version needs to be brought back here, same goes for VelocityMethods#getVersion

Reply:
Oops, must've overriden this when I pulled it